ddtrace/tracer/rules_sampler.go

```diff
@@ -154,6 +163,29 @@ func RateRule(rate float64) SamplingRule {
 	}
 }
 
+func TagsResourceRule(tags map[string]*regexp.Regexp, resource, name, service string, rate float64, targetRoot bool) SamplingRule {
+	return SamplingRule{
+		Service:        globMatch(service),
+		Name:           globMatch(name),
+		Resource:       globMatch(resource),
+		Rate:           rate,
+		TargetSpanRoot: targetRoot,
+		Tags:           tags,
+		ruleType:       SamplingRuleTrace,
+	}
+}
+
+func SpanTagsResourceRule(tags map[string]*regexp.Regexp, resource, name, service string, rate float64) SamplingRule {
+	return SamplingRule{
+		Service:  globMatch(service),
+		Name:     globMatch(name),
+		Resource: globMatch(resource),
+		Rate:     rate,
+		Tags:     tags,
+		ruleType: SamplingRuleSpan,
+	}
+}
```

Comment:
This doesn't seem to be used, shouldn't we use it to unmarshall `DD_SPAN_SAMPLING_RULES` ?

Reply:
This might not be used in the package, but is likely to be used by customers who configure rules through the code options

Reply:
Got it! Let's add doc strings then!

Reply:
already added 🙌